### ISSUE TITLE
Add sonar.coverageReportPaths setting

### ISSUE THREAD

**Issue by gjhommersom** [cake-contrib/Cake.Sonar#103]
The setting for the generic test data format is missing.

As a workaround I'm currently using inheritance to fix it but it would be nice if this worked out of the box.

```
    public class MySonarBeginSettings : SonarBeginSettings
    {
        [Argument("/d:sonar.coverageReportPaths=", true)]
        public string CoverageReportPaths { get; set; }
    }
```

**Comment by mcliment** [comment]
I could add this, as well as what's requested in #91 but I don't know exactly what's the policy when deciding to include or not sonar parameters vs using ArgumentCustomizaton. Any insights @tomstaijen?

**Comment by tomstaijen** [comment]
I typically used to add it if it was a common parameter, and not something
for a very specific tool.

**Comment by mcliment** [comment]
@tomstaijen I think that the requested parameters here, in #91 and #95 are quite reasonable. I'll make a PR with those.

**Comment by HofmeisterAn** [comment]
Is now included in version `1.1.30` (#111).
